modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/RasterLayerResponse.java

```diff
@@ -875,150 +511,74 @@ private MosaicElement createMosaic() throws IOException {
 
     /**
      * This class is responsible for putting together the granules for the final mosaic.
-     * 
+     *
      * @author Simone Giannecchini, GeoSolutions SAS
-     * 
      */
     private class MosaicProducer implements GranuleCatalogVisitor {
```

Comment:
Never mind, found a usage in RasterLayerResponse.prepareResponse, the search output in Eclipse looked it preparedResponse was a method of MosaicProdurcer instead.